x-pack/test/functional/config.js

```diff
@@ -221,6 +221,17 @@ export default async function ({ readConfigFile }) {
           kibana: [],
         },
 
+        global_canvas_read: {
```

Comment:
Since this has more than just read ability, maybe it should be renamed to something like `global_canvas_all`

Reply:
agreed. will make the changes. Thank u